format/Schema.fbs

```diff
@@ -283,6 +286,8 @@ table DictionaryEncoding {
   /// is used to represent ordered categorical data, and we provide a way to
   /// preserve that metadata here
   isOrdered: bool;
+
+  dictionaryKind: DictionaryKind;
```

Comment:
I suppose I need to spend some time looking at the sparseness / encoding proposal to understand this better. Is your thinking that alternative dictionary-like encodings would not be handled through some other metadata, and better here?

Reply:
This isn't part of the proposal but came from a paper that Ippokratis linked to (https://15721.courses.cs.cmu.edu/spring2017/papers/11-compression/p283-binnig.pdf)

Reply:
to provide more context.  It seems that in the future for ordered dictionaries having explicit numeric keys might be advantageous.

Reply:
I understand this now